src/client/components/RegisterContent.tsx

```diff
@@ -1,8 +1,9 @@
 /** @jsx jsx */
 import React, { useState } from "react";
-import { Box, Button, Flex, jsx } from "theme-ui";
+import { Box, Button, Checkbox, Flex, jsx, Label, Link, Themed } from "theme-ui";
 import { toast } from "react-toastify";
 import { connect } from "react-redux";
+import { useLocation } from "react-router-dom";
```

Comment:
Tragically `react-router-dom` _also_ has a `Link` component that we need to use for internal links.
I've mixed these up before too...

```suggestion
import { Box, Button, Checkbox, Flex, jsx, Label, Themed } from "theme-ui";
import { toast } from "react-toastify";
import { connect } from "react-redux";
import { Link, useLocation } from "react-router-dom";
```